api-extractor/src/ApiDefinitionReference.ts

```diff
@@ -0,0 +1,195 @@
+/**
+ * A scope and package name are semantic information within an API reference expression.
+ */
+export interface IScopePackageName {
+  /**
+   * The scope name of an API reference expression.
+   */
```

Comment:
Include an example. If there is no scope, say whether it's undefined or an empty string.